.eslintrc

```diff
@@ -0,0 +1,42 @@
+{
+  // https://medium.com/@dan_abramov/lint-like-it-s-2015-6987d44c5b48
+  "parser": "babel-eslint",
+
+  "env": {
+    "node": true,
+    "mocha": true,
+    "es6": true,
+  },
+
+  "rules": {
+    "max-len": [2, 120, 2],
+    "quotes": [2, "single"],
+    "complexity": [1, 10],
+    "curly": 2,
+    "default-case": 2,
+    "no-lonely-if": 2,
+    "no-multiple-empty-lines": [1, { "max": 1} ],
+    "no-nested-ternary": 2,
+    "quote-props": [2, "as-needed"],
+    "space-after-keywords": 1,
+    "space-before-blocks": 1,
+    "space-before-function-parentheses": [1, "always"], 
+    "space-in-brackets": [1, "never"],
+    "space-in-parens": [1, "never"],
+    "no-underscore-dangle": [0],
+    "no-extra-parens": 2,
+    "indent": [2, 2, {"indentSwitchCase": true}],
+    "consistent-this": [1, "that"],
+    "operator-assignment": [2, "always"],
+    "semi": [2, "always"],
+    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+    "default-case": 2,
+    "no-eq-null": 2,
+    "no-floating-decimal": 2,
+    "no-self-compare": 2,
+    "no-throw-literal": 2,
+    "radix": 2,
+    "yoda": [1, "never"],
+    "strict": [2, "global"],
```

Comment:
No need for `use strict` with babeljs..

Perhaps we need to steal some rules from: https://github.com/babel/babel/blob/master/.eslintrc